pyCGM_Single/tests/test_pycgm_angle.py

```diff
@@ -0,0 +1,96 @@
+import unittest
+import pyCGM_Single.pyCGM as pyCGM
+import numpy as np
+from parameterized import parameterized
+
+rounding_precision = 8
+
+class TestPycgmAngle(unittest.TestCase):
+
+    @parameterized.expand([
+        [[[0.09010104879445179904, 0.99590937599884910014, 0.00680557152145411237],
+          [0.99377608791559168822, -0.08945440277921079542, -0.06638521605464120512],
+          [-0.06550487076049194002, 0.01274459183355247660, -0.99777085910818641423]],
+         [[0.14362551354236074985, -0.98963110439615320502, -0.00141034441480769601],
+          [0.68540404438920177199, 0.09844473917718232769, 0.72147760135931093828],
+          [-0.71385783444225126004, -0.10458924677044478813, 0.69243633762630452111]],
+         [-9.42569150165887, 130.86647058885387, -170.3887751198432]],
+        [[[-1, 0, 0], [0, 0, 0], [0, 0, 0]],
```

Comment:
It looks like you are repeating these a few times, can you give them a variable name and put it at the top instead? Like `frame_45` so it isn't copied on each test

Reply:
I added some variables for test cases where I could. I wasn't sure how to name them well since they had a lot of similarities. For example `testcase_a` and `testcase_c` are both 90 degrees and used in `getangle_sho` and `getangle`, the difference is that `testcase_a` is 90 degrees in the first index of the result and `testcase_c` is the second index